core/Midi.cpp

```diff
@@ -124,7 +137,13 @@ void Midi::midiOutputLoop(){
 void Midi::readInputLoop(){
 	while(!gShouldStop){
 		int maxBytesToRead = inputBytes.size() - inputBytesWritePointer;
-		int ret = read(inputPort, &inputBytes[inputBytesWritePointer], sizeof(midi_byte_t)*maxBytesToRead);
+        int ret = -1;
```

Comment:
@TheTechnobear everything looks good, thanks. Let's try to stick to tabs for indentation next time!